docs/guides/main.rst

```diff
@@ -182,6 +182,11 @@ here, including the directory and file naming scheme. See
 Importing Your Library
 ----------------------
 
+The next step is to import your music files into the beets library database.
+Because this can involve moving files around, data loss is always a
```

Comment:
I'd just add a mention of the fact that beets can also modify files.

"Because this can involve moving files around as well as modifying them, data loss is always a"

Reply:
Indeed; thank you!